Assets/MixedRealityToolkit.Staging/UnityAR/README.md

```diff
@@ -0,0 +1,6 @@
+# Microsoft.MixedReality.Toolkit.Providers.UnityAR
+
+This folder contains the code to enable mobile AR, via Unity AR Foundation, on MRTK.
+
+> [!Note]
+> Please do NOT delete any file containing the text "asmdef" from this folder. Doing so will break compilation.
```

Comment:
>Doing so will break compilation. [](start = 79, length = 32)

Does this break compilation in a way that triggers a build/CI error, or is it more of a silent error?

Reply:
This is no longer relevant... Thanks for pointing it out.

Reply:
Removed